lib/atom_utils.dart

```diff
@@ -76,6 +71,15 @@ Future<String> promptUser(String prompt,
 
   Panel panel = atom.workspace.addModalPanel(item: element, visible: true);
 
+  Timer.run(() {
+    // There's a bug in Dart's JS interop where we can't see custom elements
+    // well. In order to work around it, we get the raw JS object, wrap it in a
+    // JsObject, pass that to a semantic wrapper around TextEditorElement, and
+    // then call `focused()`.
```

Comment:
This had me stumped as to why promptUser was not getting focus recently. Thanks for fixing this!